docs/user/web_data.rst

```diff
@@ -396,37 +396,37 @@ Appendix A: GeoServer Jars
 +------------------------------------------------------------+------------+
 | GeographicLib-Java-1.44.jar                                | 31693      |
 +------------------------------------------------------------+------------+
-| geomesa-accumulo-datastore-1.2.0-SNAPSHOT.jar              | 2338358    |
+| geomesa-accumulo-datastore_2.11-1.3.0-m0.jar               | 3059818    |
```

Comment:
Honestly I am not sure what the point of including all of the JARs *and* their sizes is; that's an old artifact of the README.md file this was copied from.